cypress/e2e/recieve-notice/page-request-cancel-request-triggers.cy.js

```diff
@@ -0,0 +1,260 @@
+import generateItemBarcode from '../../support/utils/generateItemBarcode';
+import InventoryInstances from '../../support/fragments/inventory/inventoryInstances';
+import PatronGroups from '../../support/fragments/settings/users/patronGroups';
+import Location from '../../support/fragments/settings/tenant/locations/newLocation';
+import Users from '../../support/fragments/users/users';
+import SearchPane from '../../support/fragments/circulation-log/searchPane';
+import CirculationRules from '../../support/fragments/circulation/circulation-rules';
+import NoticePolicyApi from '../../support/fragments/circulation/notice-policy';
+import NoticePolicyTemplateApi from '../../support/fragments/circulation/notice-policy-template';
+import NewNoticePolicy from '../../support/fragments/circulation/newNoticePolicy';
+import NewNoticePolicyTemplate from '../../support/fragments/circulation/newNoticePolicyTemplate';
+import ServicePoints from '../../support/fragments/settings/tenant/servicePoints/servicePoints';
+import getRandomPostfix from '../../support/utils/stringTools';
+import RequestPolicy from '../../support/fragments/circulation/request-policy';
+import NewRequest from '../../support/fragments/requests/newRequest';
+import Requests from '../../support/fragments/requests/requests';
+
+describe('Request notice triggers', () => {
+  let addedCirculationRule;
+  const patronGroup = {
+    name: 'groupToTestNotices' + getRandomPostfix(),
+  };
+  let userData;
+  const itemData = {
+    barcode: generateItemBarcode(),
+    title: `Instance ${getRandomPostfix()}`,
+  };
+  const testData = {
+    userServicePoint: ServicePoints.getDefaultServicePointWithPickUpLocation('autotestReceiveNotice', uuid()),
+    ruleProps: {},
+  };
+  const createNoticeTemplate = (noticeName) => {
+    return {
+      name: `Autotest_${getRandomPostfix()}_${noticeName}`,
+      description: 'Created by autotest team',
+      category: 'Request',
+      subject: `Autotest_${getRandomPostfix()}_${noticeName}`,
+      body: 'Test email body {{item.title}} {{loan.dueDateTime}}',
+    };
+  };
+  const noticeTemplates = {
+    pageRequest: createNoticeTemplate('Page_request'),
+    cancelRequest: createNoticeTemplate('Cancel request'),
+  };
+  const searchResultsData = (description) => {
+    return {
+      userBarcode: userData.barcode,
+      itemBarcode: itemData.barcode,
+      object: 'Notice',
+      circAction: 'Send',
+      // TODO: add check for date with format <C6/8/2022, 6:46 AM>
+      servicePoint: testData.userServicePoint.name,
+      source: 'System',
+      desc: description,
+    };
+  };
+  const checkNoticeIsSent = (checkParams) => {
+    SearchPane.searchByUserBarcode(userData.barcode);
+    SearchPane.findResultRowIndexByContent(checkParams.desc).then((rowIndex) => {
+      SearchPane.checkResultSearch(checkParams, rowIndex);
+    });
+  };
+  const noticePolicy = {
+    name: `Autotest ${getRandomPostfix()} Overdue fine, returned`,
```

Comment:
let's use the method getTestEntityValue(Value) from utils/stringTools.js

Reply:
done